.github/workflows/eerelease.yml

```diff
@@ -0,0 +1,3 @@
+name: Release Excution Environments
+on:
+
```

Comment:
I think this should not be part of this PR.

Reply:
Yes,  I pushed it by mistake. Removed it now.